src/st/np3/e_misc.c

```diff
@@ -285,7 +324,7 @@ void func_801C1848(void) {
 
 #include "../clut_lerp.h"
 
-void func_801C2598(s16 sfxId) {
+void func_801916C4(s16 sfxId) {
```

Comment:
I'm thinking these renames are being done to align the names among multiple overlays? We might want to pick actual names if the address doesn't match the function name anymore

Reply:
I was thinking `PlaySfx3D` specifically for this one. But I have no idea what the others are doing. I was going to replace this one with `#include "../e_misc.h` if it was not for `EntityHeartDrop`